MOKApp/app/src/main/java/mok/it/app/mokapp/fragments/AllProjectsListFragment.kt

```diff
@@ -215,9 +216,12 @@ class AllProjectsListFragment :
                     loadImage(ivImg, model.icon, callback)
                 }
 
-                if (userModel.collectedBadges.contains(model.id)) {
+                if (userModel.projectBadges.contains(model.id)) {
```

Comment:
A gradient eddig azt jelentette, hogy az illető megyszerezte a mancsot, tehát logikusan most azt kellene jelentenie, hogy teljesítette a projektet. A feltétel viszont az, hogy legyen benne a user projectbadges-jében az adott kulcs. Ez akkor is megtörténik, ha a usert jelentkeztetik a projektre, így jelenleg a feljelentkezett projektekeknek a háttere lesz gradient. Ez volt a terv?

Reply:
Akkor legyen gradient szerintem, ha fel van jelentkezve. Fontosabb lehet azt kiemelni, hogy mire van felcsatlakozva.